src/amp-story-player/amp-story-player-impl.js

```diff
@@ -985,11 +985,18 @@ export class AmpStoryPlayer {
   /**
    * Navigates stories given a number.
    * @param {number} storyDelta
+   * @param {number=} pageDelta
    */
-  go(storyDelta) {
-    if (storyDelta === 0) {
+  go(storyDelta, pageDelta = 0) {
+    if (storyDelta === 0 && pageDelta === 0) {
       return;
     }
+
+    if (storyDelta === 0 && pageDelta !== 0) {
+      this.selectPage_(pageDelta);
```

Comment:
We should not duplicate the call to `selectPage_`, this makes the logic a bit harder. Can we gate the `this.show()` and its logic on `storyDelta` being different than 0 instead?